src/pages/AddProposal/AddProposal.types.ts

```diff
@@ -16,7 +16,8 @@ export type AddProposalFields = {
 
 export type AddProposalProps = {
   isOpenDialog: boolean;
-  eligibleToAdd: boolean;
+  reasonToBlock?: string;
+  onGovernorChange?: (gov: string) => void;
```

Comment:
I think this prop could be required

```suggestion
  onGovernorChange: (gov: string) => void;
```